static/build/.tmp_update/script/easynetplay/netplay_server.sh

```diff
@@ -193,11 +193,14 @@ build_infoPanel() {
     local title="$1"
     local message="$2"
     
-    infoPanel --title "$title" --message "$message" --auto
+    infoPanel --title "$title" --message "$message" --persistent &
+    sleep 0.5
```

Comment:
You should move the `sleep` after `sync` - to ensure infoPanel has closed before continuing

Reply:
Resolved in https://github.com/OnionUI/Onion/pull/969/commits/9562a6583dbcf500e0a78bd976ebe49fc00214b7